crates/db_schema/src/source/email_verification.rs

```diff
@@ -0,0 +1,18 @@
+use crate::{newtypes::LocalUserId, schema::email_verification};
+
+#[derive(Queryable, Identifiable, Clone)]
+#[table_name = "email_verification"]
+pub struct EmailVerification {
+  pub id: i32,
+  pub local_user_id: LocalUserId,
+  pub email: String,
```

Comment:
Hrm.... email is already on the local_user table, so I don't think it should be required here.